src/Modals/ModalMessage.tsx

```diff
@@ -11,6 +11,10 @@ export interface ModalMessageProps {
   children: string;
   /** Allows you to pass in a variant property from the VARIANT enum */
   variant: VARIANT;
+  /** The aria role of the message displayed */
```

Comment:
Can we add a brief note about what value(s) can/should be used here, and what the default value is?

Reply:
Definitely. This was added in 8625dd3.